config.js

```diff
@@ -1 +1 @@
-{"netbox_url":"","netbox_token":"","ui":1,"id":"","endpoint":"","flows":"0"}
```

Comment:
why uploading the config.js ?

Reply:
Because it's not in the .gitignore file?